scripts/deploy_subscription_manager.py

```diff
@@ -25,10 +41,10 @@ def main(account_name : str = 'test'):
     subscription_manager_logic = deployer.deploy(SubscriptionManager)
     calldata = subscription_manager_logic.initialize.encode_input(INITIAL_FEE_RATE)
     transparent_proxy = oz.TransparentUpgradeableProxy.deploy(
-        subscription_manager_logic.address,
-        proxy_admin.address,
-        calldata,
-        {'from': deployer})
+        subscription_manager_logic.address, proxy_admin.address, calldata, {"from": deployer}
+    )
 
-    subscription_manager = Contract.from_abi("SubscriptionManager", transparent_proxy.address, subscription_manager_logic.abi, owner=None)
+    subscription_manager = Contract.from_abi(
+        "SubscriptionManager", transparent_proxy.address, subscription_manager_logic.abi, owner=None
+    )
```

Comment:
I merged a few PRs today, so this part should go after rebasing.

Reply:
rebased